l3kernel/l3doc.dtx

```diff
@@ -2172,17 +2180,20 @@ and all files in that bundle must be distributed together.
 %    \begin{macrocode}
 \cs_new_protected:Npn \@@_names_get_seq:nN #1#2
   {
-    \__kernel_tl_set:Nx \l_@@_tmpa_tl { \tl_to_str:n {#1} }
     \bool_if:NTF \l_@@_names_verb_bool
       {
         \seq_clear:N #2
-        \seq_put_right:NV #2 \l_@@_tmpa_tl
+        \seq_put_right:Ne #2 { \tl_to_str:n {#1} }
```

Comment:
Would `:No` be quicker?

Reply:
I would think so (and in some sense more accurate in my opinion)

Reply:
And lucky the doc of `\tl_to_str:n` says "The base function requires only a single expansion."